recipes/aiosmtplib/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "aiosmtplib" %}
+{% set version = "1.0.2" %}
+{% set sha256 = "c9d77f9f7a7d6fdb8db29b412fa3971a13c01be1042f47a0e0a28522c9a494ff" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [py<35]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - aiosmtplib
+
+about:
+  home: https://github.com/cole/aiosmtplib
+  license: MIT
+  license_family: MIT
```

Comment:
Manually include the `license_file`:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

Please add an issue upstream asking that the license be distributed with the source distribution and add the url to the issue as a comment here as a reminder to check if it has been added in a future release.

Reply:
Done